### ISSUE TITLE
[FEATURE REQUEST]Configurable Sticker Size in conversations

### ISSUE THREAD

**Issue by KaoticoFanegas** [telegramdesktop/tdesktop#735]
I think it would be nice to allow users to configure the drawing size of the stickers in the conversation window, I've noticed a sensible difference between Telegram Desktop and Telegram Android ( Way bigger in TDesktop). Such big stickers can draw the last lines of conversation out of the window when working in the smallest size possible.

Thanks for your time, you are doing great!

**Comment by vladimir-sol** [comment]
I'd like to +1 this.

Stickers, while technically being just images (a point made by @telegramdesktop), UNLIKE regular photos or images promote spamming them. Currently, on my 11" Air, I can only see 1 sticker + 4 (four!) lines of messages in a reasonably sized telegram window, and there're stickers flying around constantly.
![screen shot 2015-09-24 at 7 55 26 pm](https://cloud.githubusercontent.com/assets/3204795/10072817/1d96551a-62f7-11e5-9a30-a78804188852.png)

**Comment by STrRedWolf** [comment]
I'm also for this.  Damn things are huge both on iOS and on the desktop versions.

**Comment by auchri** [comment]
@vladimir-sol The screenshot you added is not from Telegram Desktop.

**Comment by thexile** [comment]
Agree. The sticker size on desktop telegram is a little too big.

**Comment by elessarperm** [comment]
It would be great if we could configure sticker size in application settings!

**Comment by anibyl** [comment]
+1
We need configurable sticker size.

**Comment by Denizhen** [comment]
+1

**Comment by luvini** [comment]
![telegram](https://user-images.githubusercontent.com/32948452/31805225-f06c0c42-b53d-11e7-88cc-c5e2027e560e.png)
comparison between versions of Telegram in Android, Desktop, WEB Mobile, WEB Desktop

i believe the stickers on both Desktop and WEB Desktop should have height 120px by default, and allow the user to change it, just like text size in Android (maybe put sizes multiple of 8, begining at 64, up to 512)

it's important to take into account the size ratio between TEXT and STICKER

![telegram2](https://user-images.githubusercontent.com/32948452/31805767-3b780886-b542-11e7-9a25-3477527c3f72.png)

**Comment by PeterMX** [comment]
Add this feature please!
if I made a pull request for this would you merge it?

**Comment by ArmeF97** [comment]
@PeterMX Try, I think this may be game-changing

**Comment by NexonSU** [comment]
You can change it for yourself in source code:
![image](https://user-images.githubusercontent.com/5037786/53284219-108db580-3773-11e9-9f2d-284993d33542.png)

**Comment by ArmeF97** [comment]
@nexonsu yeah, but we need a slider for the size in the settings

**Comment by STrRedWolf** [comment]
The idea is to expose this in the Settings UI and have a reasonable range.

**Comment by ArmeF97** [comment]
@strredwolf Good! Can you male a PR out of this?

**Comment by sirian** [comment]
pls anyone make a PR!!! stickers occupies tooo much space

**Comment by KartulUdus** [comment]
Yes please, this pikachu get's too upscaled and needs more pixels as sticker

`client.telegram.sendSticker(msg.job.target, `https://cdn.discordapp.com/attachments/533986578763087872/559693087325290506/pokemon_icon_025_00.webp`, {disable_notification: true})`

=>

![image](https://user-images.githubusercontent.com/23346150/54915221-54edac00-4eff-11e9-8700-c4846fa7176d.png)

**Comment by sqliuchang** [comment]
We need an option to set the size of stickers and they are really too large in desktop version.

**Comment by qzi** [comment]
PR

**Comment by willww64** [comment]
Please add this feature!The default sticker size is so giant and we have to scoll back more to see history message. And at public area, someone beside you can easily see the giant stickers and it's embarassing sometimes.

**Comment by mpg-truong-thanhtung** [comment]
I'm waiting this one a long time, please do it.

**Comment by sqliuchang** [comment]
> I'm waiting this one a long time, please do it.

You can try this https://github.com/Forkgram/tdesktop

**Comment by Aokromes** [comment]
in case anyone want PR it, this is the commit adding it:
https://github.com/Forkgram/tdesktop/commit/8be8fb56b359fa563c5d0aa1e9d0f546fd9da7d2

**Comment by STrRedWolf** [comment]
Registering continued interest with this.  Please add in mainline!

**Comment by STrRedWolf** [comment]
BTW, apologies for the delay (things have been busy on my end).  What is "PR" in this context?

**Comment by ArmeF97** [comment]
PR means Pull Request.
It means that you write the code and submit a pull request, so the owner of the repo can accept/refuse it as part of the repo

**Comment by ilya-fedin** [comment]
> in case anyone want PR it, this is the commit adding it:

You know, preston doesn't like options, even if someone PR this, preston will just close the PR.

**Comment by IneOtherGuys** [comment]
Bring up this post, this needs to become a thing. With all the uproar with Whatsapp this year I'm considering using Telegram as my main chat client but the huge stickers are definitely annoying in the desktop app. Not because I don't like them, on the contrary, because I love to use them, but now sending two or three feels like I'm destroying the conversation. It would be so simple to add an option to cap the max resolution.

**Comment by ars4l4n** [comment]
> > in case anyone want PR it, this is the commit adding it:
> 
> You know, preston doesn't like options, even if someone PR this, preston will just close the PR, just like it was with #7822.

if he doesn't like options then he should at least consider the argument of consistency, especially between the web version and the Desktop version

**Comment by thexdd** [comment]
Is there at least any way I could reduce stickers size?

**Comment by Johanfer12** [comment]
Any changes?

**Comment by Aokromes** [comment]
comming on next version.

**Comment by ilya-fedin** [comment]
`"sticker-size": 192` in `tdata/experimental_options.json`
